src/core/event/index.js

```diff
@@ -217,7 +219,15 @@ export function Events(Base) {
      * @void
      */
     #btn(el) {
-      const toggle = _ => dom.body.classList.toggle('close');
+      const toggle = _ => {
+        dom.body.classList.toggle('close');
+
+        const isClosed = isMobile
+          ? dom.body.classList.contains('close')
+          : !dom.body.classList.contains('close');
```

Comment:
This is so confusing without going and learning how the class works.

Reply:
@trusktr --

This application of the `close` class is confusing because the existing implementation (which predates all of the current maintainers AFAIK) is confusing. Refactoring the sidebar visibility logic is outside of the scope of this PR which is focused on adding `aria` labels for accessibility purposes.

FYI:

The `close` class means two opposite things based on the viewport width *when the page was loaded*. It is only applied to the `<body>` tag when the user clicks the sidebar button. Since the sidebar menu is hidden on mobile by default but displayed on desktop by default:

When a Docsify site loads, the lack of a `close` class on `<body>` indicates...
- On mobile the sidebar is hidden
- On desktop the sidebar is visible

When the sidebar toggle button is clicked and a `close` class is added to `<body>`, it indicates...
- On mobile the sidebar is visible
- On desktop the sidebar is hidden

Besides being confusing for maintainers, the pre-existing implementation also has the downside of inverting the user-specified sidebar visibility menu state when resizing the viewport between desktop and mobile widths. This is because `isMobile` is calculated only one time (when the page initially loads) instead of when the viewport size is adjusted. For example, if you load a page at a desktop resolution and hide the sidebar, resizing the viewport to a mobile resolution will cause the sidebar to appear.

We should clean this up, but that is a discussion to be had outside of this PR.